pyteal/ast/box_test.py

```diff
@@ -151,10 +151,10 @@ def test_box_get():
 
     expected = pt.TealSimpleBlock(
         [
-            pt.TealOp(name_arg, pt.Op.byte, '"eineName"'),
-            pt.TealOp(expr, pt.Op.box_get),
-            pt.TealOp(expr.output_slots[1].store(), pt.Op.store, expr.output_slots[1]),
-            pt.TealOp(expr.output_slots[0].store(), pt.Op.store, expr.output_slots[0]),
+            pt.TealOp(None, pt.Op.byte, '"eineName"'),
+            pt.TealOp(None, pt.Op.box_get),
+            pt.TealOp(None, pt.Op.store, expr.output_slots[1]),
+            pt.TealOp(None, pt.Op.store, expr.output_slots[0]),
```

Comment:
This isn't necessary to change, but since the origin `Expr` is ignored when comparing under `pt.TealComponent.Context.ignoreExprEquality()`, it's unnecessary to populate it

Reply:
I see, `ignoreExprEquality` ignores the `expr`s that are replaced to `None` by you, right?

Reply:
That's right